make/test.mk

```
@@ -175,6 +179,25 @@ else
 	$(MAKE) print-local-debug-info  HOST_NS=${HOST_NS} MEMBER_NS=${MEMBER_NS} MEMBER_NS_2=${MEMBER_NS_2} REGISTRATION_SERVICE_NS=${REGISTRATION_SERVICE_NS}
 endif
 
+.PHONY: check-go-junit-report
+
+check-go-junit-report:
+	PATH=${PATH}:$(go env GOPATH)/bin
+	@command -v go-junit-report >/dev/null 2>&1 || { echo "go-junit-report is not installed. Installing..."; go install github.com/jstemmer/go-junit-report/v2@latest; }
+	@echo "go-junit-report version:" && go-junit-report -version
+
+.PHONY: generate-report
+generate-report:
+	@echo "Generating report"
+ifneq ($(OPENSHIFT_BUILD_NAMESPACE),) 
+	mkdir -p  ${ARTIFACT_DIR}/${REPORT_PORTAL_DIR}
+	PATH=${PATH}:$(go env GOPATH)/bin
+	go-junit-report > ${ARTIFACT_DIR}/${REPORT_PORTAL_DIR}/${REPORT_NAME}
```

my bad, maybe this picks up the updated `PATH`
```suggestion
	PATH=${PATH}:$(go env GOPATH)/bin go-junit-report > ${ARTIFACT_DIR}/${REPORT_PORTAL_DIR}/${REPORT_NAME}
```